onnx_tf/handlers/backend/quantize_linear.py

```diff
@@ -0,0 +1,26 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+
+
+@onnx_op("QuantizeLinear")
+class QuantizeLinear(BackendHandler):
+
+  @classmethod
+  def version_10(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    y_scale = kwargs["tensor_dict"][node.inputs[1]]
+    y_zero_point = tf.zeros((), tf.int8)
```

Comment:
The default dtype is uint8, according to https://github.com/onnx/onnx/blob/master/docs/Operators.md#QuantizeLinear